### PR TITLE
Update change account info to reflect MyRack updates

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2abf4ffc6c/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d1d7478290/how-to/)

File: content/general/change-account-information.md [hunk 1]
```diff
@@ -5,17 +5,27 @@ title: Change account information
 type: article
 created_date: '2015-06-18'
 created_by: David Hendler
-last_modified_date: '2018-03-13'
+last_modified_date: '2018-05-22'
```
Comment (cyrichardson) [inline content/general/change-account-information.md]: 2018-05-24

File: content/general/change-account-information.md [hunk 2]
```diff
@@ -5,17 +5,27 @@ title: Change account information
...
 
 This article describes the various administrative account changes that you can request and the appropriate steps needed to complete them.
 
-**Note:** If you are a Dedicated Hosting customer, you can request all of the forms described in this article from your account manager.
+### Dedicated customers
 
-### Service transform form requirements
+Dedicated customers can download the **Primary Contact Change** form in the [MyRackspace Control Panel](https://my.rackspace.com). For instructions on how to find the form, see the **Dedicated customers** section of [Change to a new primary contact](/how-to/change-to-a-new-primary-contact).
+
+If you are the current primary contact for your account, you can update the primary contact information by selecting and editing the **Primary Contact Informtion** in your **Account Settings**.
```
Comment (cyrichardson) [inline content/general/change-account-information.md]: **Primary Contact Information**

File: content/general/change-account-information.md [hunk 3]
```diff
@@ -5,17 +5,27 @@ title: Change account information
...
+
+The **Service Transfer** form can be requested from your account manager.
```
Comment (cyrichardson) [inline content/general/change-account-information.md]: You request the **Service Transfer** form from your account manager.

File: content/general/change-account-information.md [hunk 4]
```diff
@@ -5,17 +5,27 @@ title: Change account information
...
+
+### All other customers
+
+All other customers can find links to the forms described in this article log in to the [Cloud Control Panel](http://mycloud.rackspace.com) and select **Account** > **Docs and Forms**.
```
Comment (cyrichardson) [inline content/general/change-account-information.md]: article by logging in to the [Cloud Control Panel](http://mycloud.rackspace.com) and selecting **Account** > **Docs and Forms**.

File: content/general/change-account-information.md [hunk 5]
```diff
@@ -5,17 +5,27 @@ title: Change account information
...
+
+### Service transfer form requirements
```
Comment (cyrichardson) [inline content/general/change-account-information.md]: Service Transfer

File: content/general/change-account-information.md [hunk 6]
```diff
@@ -48,10 +58,10 @@ In the **Cloud Account Number** and the **New Cloud Account Name** columns, prov
 
 ### Best practices for legal change forms
 
-The both the service transfer and primary contact transfer forms have many required fields that must be
+Both the service transfer and primary contact transfer forms have many required fields that must be
```
Comment (cyrichardson) [inline content/general/change-account-information.md]: Service Transfer and Primary Contact Transfer forms have many required fields that you must

(Make sure that Service Transfer and Primary Contact Transfer are initial caps throughout this article and that form or forms is always lowercase throughout the article.)

On L52  **Page 3 - Consolidated and/or Linked Cloud Accounts (Exhibit A)** 
   Add space around slash - and / or - to be consistent with how the slash is handled above this line.

File: content/general/change-account-information.md [hunk 7]
```diff
@@ -48,10 +58,10 @@ In the **Cloud Account Number** and the **New Cloud Account Name** columns, prov
...
 completed before the Billing team can process the forms. After you have
```
Comment (cyrichardson) [inline content/general/change-account-information.md]: complete

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-20c5d03040/how-to/)

File: content/general/change-account-information.md [hunk 8]
```diff
@@ -5,17 +5,27 @@ title: Change account information
...
 
-**Note:** If you are a Dedicated Hosting customer, you can request all of the forms described in this article from your account manager.
+### Dedicated customers
 
-### Service transform form requirements
+Dedicated customers can download the **Primary Contact Change** form in the [MyRackspace Control Panel](https://my.rackspace.com). For instructions on how to find the form, see the **Dedicated customers** section of [Change to a new primary contact](/how-to/change-to-a-new-primary-contact).
+
+If you are the current primary contact for your account, you can update the primary contact information by selecting and editing the **Primary Contact Information** in your **Account Settings**.
+
+You request the **Service Transfer** form from your account manager.
```
Comment (catlook) [inline content/general/change-account-information.md]: Request the **Service Transfer** form from your account manager.

File: content/general/change-account-information.md [hunk 9]
```diff
@@ -5,17 +5,27 @@ title: Change account information
...
+
+### All other customers
+
+All other customers can find links to the forms described in this article log in to the [Cloud Control Panel](http://mycloud.rackspace.com) and selecting **Account** > **Docs and Forms**.
```
Comment (catlook) [inline content/general/change-account-information.md]: ... this article by logging in to the ...

File: content/general/change-account-information.md [hunk 10]
```diff
@@ -39,7 +49,7 @@ The Billing team has strict requirements for processing the Service Transfer for
 Do not fill out the **RACKSPACE US, INC.** section. A Rackspace
 representative completes this section.
 
-**Page 3 - Consolidated and/or Linked Cloud Accounts (Exhibit A)**
+**Page 3 - Consolidated and / or Linked Cloud Accounts (Exhibit A)**
```
Comment (catlook) [inline content/general/change-account-information.md]: [suggestion: drop the slash and pick either "and" or "or"]

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5e55cd1c43/how-to/).

**Pull request merged**